dev_guide/builds/triggering_builds.adoc

```diff
@@ -90,11 +93,97 @@ $ curl -H "X-GitHub-Event: push" -H "Content-Type: application/json" -k -X POST
 The `-k` argument is only necessary if your API server does not have a properly
 signed certificate.
 
+[[gitlab-webhooks]]
+=== GitLab Webhooks
+
+https://docs.gitlab.com/ce/user/project/integrations/webhooks.html[GitLab webhooks]
+handle the call made by GitLab when a repository is updated. As with GitHub
+triggers, you must specify a `secret`. The following example is
+a trigger definition YAML within the `BuildConfig`:
+
+[source,yaml]
+----
+type: "GitLab"
+gitlab:
+  secret: "secret101"
+----
+
+The payload URL is returned as the GitLab Webhook URL by the `describe` command
```

Comment:
Perhaps clarify that `describe` is a command off of our CLI ... i.e. the `oc` executable.